### DISCUSSION TITLE
Карта на весь экран при popup: true

### DISCUSSION THREAD

**Discussion by VychuzhaninovNA** [cdek-it/widget#51] [Q&A]
Добрый день.
Карта открывается по нажатию кнопки, стоит popup: true 
Не получается стилями сделать карту на весь экран. Пробовал стилями перебить ширину и высоту, не получается.
Настроек в виджете по размерам карты тоже не нашел.
Подскажите, есть ли решение?

**Comment by vermorag** [comment]
Добрый день. Необходимо для попапа перебить стили высоты и ширины самого виджета внутри попапа. Обращу внимание, что они все идут с префиксом !important для совместимости с остальными стилями сайта.

Можете подсказать бизнес процесс в котором требуется открывать карту именно на весь экран? Это мобильное устройство или десктоп?

**Reply by vermorag** [reply]
Яндекс реализовал функцию "открыть на весь экран" - подходит ли она в данном случае? https://codesandbox.io/p/sandbox/733kvt?file=%2Fvanilla%2Findex.html

**Reply by VychuzhaninovNA** [reply]
Как это совместить с виджетом СДЭК?
Не соображу

**Reply by piton-zel** [reply]
Никак, это просто отдельно открывающаяся карта яши будет. Только игра стилями, но там тоже есть интересные моменты.

**Comment by piton-zel** [comment]
Название стилей только скорее всего другие будут, но суть думаю понятна
```
.cdek-map {
  // Основное окно
  div.cdek-1rkrs0 {
    border-radius: 0 !important;
    position: absolute !important;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    height: 100% !important;
    width: 100% !important;

    // Область карты
    .cdek-c66dnq {
      width: 100% !important;
      min-width: 100% !important;
      height: 100% !important;
      min-height: 100% !important;

      .cdek-32nljf {
        height: 100% !important;
        min-height: 100% !important;
      }
    }
  }
}
```
